### ISSUE TITLE
codecov yaml not located

### ISSUE THREAD

**Issue by derhally** [codecov/codecov-exe#59]
When running on Linux, the CodeCover.Tool cannot locate the codecov.yaml file.

This is because the Yaml class is assuming paths use the \ path separator.

**Comment by AdmiringWorm** [comment]
@derhally we officially don't support linux right now, while there are plans to do that, there are several issues that prevent that support right now (path separator being only one of the issues).

**Comment by derhally** [comment]
Ok.  Are you accepting PRs to support running on other platforms?

**Comment by AdmiringWorm** [comment]
Yes I am, however, before I can merge them I need to update the unit tests to support other platforms. (This is currently in progress, and needs to be done before merging anything).

**Comment by AdmiringWorm** [comment]
Your changes is now been merged into the branch I'm working on for converting the unit tests to support non-windows platforms.

I still have some issues that I need to resolve before I can push out a new release. (No ETA on when this will happen, unfortunately).

**Comment by AdmiringWorm** [comment]
However, if you wish to test it out immediately, you can grab the nupkg for Cake.Tool on our build server: https://ci.appveyor.com/project/AdmiringWorm/codecov-exe/build/job/t75m6x2q5rgbencx/artifacts

**Comment by AdmiringWorm** [comment]
Codecov 1.5.0 is now officially released, and is available on nuget.
